phoenix-core/src/main/java/org/apache/phoenix/coprocessor/DropColumnMutator.java

```diff
@@ -268,7 +272,19 @@ public MetaDataMutationResult validateAndAddMetadata(PTable table,
             }
 
         }
-        tableMetaData.addAll(additionalTableMetaData);
+        if (isDroppingColumns) {
```

Comment:
The changes here looks exactly same as the changes in AddColumnMutator. Can we create a helper method which accepts table and additionalTableMetaData as argument ?

Reply:
It's not really general-purpose enough to put in a standalone Util method in MetaDataUtil, and since it's only 3 lines repeated in 2 places I figured the minor violation of DRY (Don't Repeat Yourself) wasn't worth working around, since all alternatives I could think of seemed a little ugly. 

For example, I could put a static utility method in one of the mutators, but then one depends on the other which couples classes that shouldn't be...